src/handlers/ground-truths/find-ground-truths.ts

```diff
@@ -19,5 +18,33 @@ async function findCodeReviewTruths(context: Context, params: CodeReviewAppParam
   const systemMsg = createGroundTruthSysMsg(systemMsgObj);
 
   const truths = await completions.createGroundTruthCompletion(context, params.taskSpecification, systemMsg);
-  return validateGroundTruths(truths);
+  return validateGroundTruths(truths, context.logger);
+}
+
+function validateGroundTruths(truthsString: string | null, logger: Context["logger"]): string[] {
+  let truths;
+  if (!truthsString) {
+    throw logger.error("Failed to generate ground truths");
+  }
+
+  try {
+    truths = JSON.parse(truthsString);
```

Comment:
shouldn't `parseLooseJson` be used because this is from the LLM?

Reply:
no because this time around, the llm returns a simple array of 10 elements which doesnt contain the curly brackets, they only contain square brackets